lib/oli_web/live/workspaces/student.ex

```diff
@@ -81,8 +78,9 @@ defmodule OliWeb.Workspaces.Student do
      )}
   end
 
-  def handle_params(params, _uri, socket),
-    do: {:noreply, assign(socket, params: decode_params(params))}
+  def handle_params(params, _uri, socket) do
+    {:noreply, assign(socket, params: decode_params(params))}
+  end
```

Comment:
"I found it difficult to keep a relevant function in one line while adding inspections because we need to break it into multiple lines, and then return to a single line after finishing the inspection. I'll keep this pattern for now and see how that goes.